common/reviews/api/terminal.api.md

```diff
@@ -4,9 +4,20 @@
 
```

Comment:
With your improvements here, and considering that we're doing a MAJOR version bump, I would suggest to graduate everything in this file to `@public`.

Reply:
This isn't a major bump of the `@rushstack/terminal` package. It is of NCL.